src/cascadia/TerminalApp/AppKeyBindings.idl

```diff
@@ -17,6 +17,7 @@ namespace TerminalApp
         CopyTextWithoutNewlines,
         PasteText,
         NewTab,
+        Dropdown,

```

Comment:
Can we rename this (and everything else) `ToggleNewTabDropdown`? We may have multiple dropdown-like things in the future, I don't want to back us into a corner by naming this an unqualified "dropdown".

Reply:
Okay let's not call it `Toggle` but the rest of my comment stands

Reply:
Is `OpenNewTabDropdown` good?

Reply:
Seems _perfect_ to me :)